### DISCUSSION TITLE
Configuring the log message template, duplicate timestamps

### DISCUSSION THREAD

**Discussion by jeffreycoxdoosan** [nuke-build/nuke#898] [Q&A] [answered]
Is there some way to configure the log messages in 6.0+ to remove the time stamps? I see that there are some CI environments where a different template is hard-coded, but that's not the case for Jenkins, and so we get output like this on our jobs:

16:38:09  ​
16:38:09  ╬═════════════════
16:38:09  ║ ZipArtifacts
16:38:09  ╬════════
16:38:09  ​
16:38:09  16:37:51 [INF] Creating directory C:\jenkins\workspace\BuildTest_feature_package-update\artifacts...
16:38:09  16:37:51 [INF] Creating Zip File: C:\jenkins\workspace\BuildTest_feature_package-update\artifacts\

**Comment by matkoch** [comment, marked as answer]
It needs to be configured for Jenkins similar to here: https://github.com/nuke-build/nuke/blob/e0bf2511c59bdf2e871b039989ec8222eee3437a/source/Nuke.Common/CI/TeamCity/TeamCity.Theming.cs#L14

**Reply by jeffreycoxdoosan** [reply]
I take that to mean "not without a pull request"? I don't see any way to override the output template without modifying the nuke source.

**Reply by matkoch** [reply]
yea that's right

**Reply by jeffreycoxdoosan** [reply]
Added pull request https://github.com/nuke-build/nuke/pull/909
